CONTRIBUTING.md

```diff
@@ -19,13 +24,17 @@ In the root you'll find `build.js`, used to build the site.
 Usage is as follows:
 
 ```shell
+# Install packages
+npm install
```

Comment:
Do we want to use `npm install` or `npm ci`?

Reply:
I'd say `install`, since that will generally be faster (as `ci` first removes the entire `node_modules`, always forcing a full reinstall).

Reply:
Okay, but a new contributor generally _doesn't_ have `node_modules` right? And `npm install` might cause unexpected or undesired updates to your `package-lock.json`.

Reply:
> a new contributor generally doesn't have node_modules right?

That's true yeah, but for subsequent installs it can make a significant difference.

> npm install might cause unexpected or undesired updates to your package-lock.json

It really shouldn't, unless we messed up somewhere :sweat_smile: Cause that would mean we edited the `package.json` without making sure the lock is also updated.

Anyway, either option is fine with me, just wanted to point out the potential drawbacks :)